src/utils.py

```diff
@@ -0,0 +1,69 @@
+import grp
+import logging
+import os
+import pwd
+import subprocess
+import tempfile
+
+logger = logging.getLogger(__name__)
+CA_CERT_PATH = "/usr/local/share/ca-certificates/{}.crt".format("nginx-server")
```

Comment:
No need for a format string if it's all constant.

```suggestion
CA_CERT_PATH = "/usr/local/share/ca-certificates/nginx-server.crt"
```